djstripe/models.py

```diff
@@ -676,74 +497,31 @@ def extend(self, delta):
         self.customer.sync_current_subscription()
 
 
-class Invoice(StripeObject):
-    stripe_api_name = "Invoice"
+class Invoice(StripeInvoice):
+    objects = models.Manager()
 
     customer = models.ForeignKey(Customer, related_name="invoices")
-    attempted = models.NullBooleanField()
-    attempts = models.PositiveIntegerField(null=True)
-    closed = models.BooleanField(default=False)
-    paid = models.BooleanField(default=False)
-    period_end = models.DateTimeField()
-    period_start = models.DateTimeField()
-    subtotal = models.DecimalField(decimal_places=2, max_digits=7)
-    total = models.DecimalField(decimal_places=2, max_digits=7)
-    date = models.DateTimeField()
-    charge = models.CharField(max_length=50, blank=True)
 
     class Meta:
         ordering = ["-date"]
 
-    def __str__(self):
-        return "<total={total}, paid={paid}, stripe_id={stripe_id}>".format(total=self.total, paid=smart_text(self.paid), stripe_id=self.stripe_id)
-
-    def retry(self):
-        if not self.paid and not self.closed:
-            inv = self.api_retrieve()
-            inv.pay()
-            return True
-        return False
-
-    def status(self):
-        if self.paid:
-            return "Paid"
-        if self.closed:
-            return "Closed"
-        return "Open"
-
     @classmethod
     def sync_from_stripe_data(cls, stripe_invoice, send_receipt=True):
         c = Customer.objects.get(stripe_id=stripe_invoice["customer"])
```

Comment:
This is already merged to the 162 branch (that, is my copy of 162, pushed to my github). For simplicty sake I will do it again here and then we will have a conflict in 162. That's okey dokey with me either way